.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Can you remove this? This is an IDE thing that you should have in your machine's global gitignore. It'll save you some keystrokes in the future, trust me :smile: